lib/driver-cli.js

```diff
@@ -0,0 +1,250 @@
+import _ from 'lodash';
+import path from 'path';
+import { KNOWN_DRIVERS, validateDriversYaml, writeInstalledDriver,
+         INSTALL_TYPE_LOCAL, INSTALL_TYPE_NPM, INSTALL_TYPE_GIT,
+         INSTALL_TYPE_GITHUB } from './drivers';
+import { getLatestVersion, installPackage, linkPackage } from './npm';
+import { errAndQuit, log, spinWith } from './cli-utils';
+import { util } from 'appium-support';
+
+let INSTALLED_DRIVERS;
+
+async function runDriverCommand (args) {
+  // TODO driver.json file should be locked while any of these commands are
+  // running to prevent weird situations
+  let jsonResult = null;
+  const {json, appiumHome} = args;
+  const logFn = (msg) => log(json, msg);
+  try {
+    INSTALLED_DRIVERS = await validateDriversYaml(appiumHome, logFn);
```

Comment:
I wouldn't focus on the format here, but rather on the business logic. how about changing the func name to  `validateDriversConfig` ?